modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/internal/info/collection/provider/test/ManyToManyObjectRelationshipInfoCollectionProviderTest.java

```diff
@@ -159,6 +162,58 @@ public void testManyToManyObjectRelationshipRelatedInfoCollectionProvider()
 		_assertRelatedInfoCollectionProvider(
 			_objectDefinition2, objectDefinition2ObjectEntry,
 			objectDefinition1ObjectEntry1, objectDefinition1ObjectEntry2);
+
+		Map<String, ServiceRegistration<?>> serviceRegistrationMap =
+			_objectRelationshipLocalService.getServiceRegistrations();
+
+		ServiceRegistration<?>
+			objectRelationshipDefinition1ServiceRegistration =
+				serviceRegistrationMap.get(
+					StringBundler.concat(
+						_objectRelationship.getCompanyId(), StringPool.POUND,
+						_objectRelationship.getObjectRelationshipId()));
+
+		objectRelationshipDefinition1ServiceRegistration.unregister();
+
+		serviceRegistrationMap.remove(
+			StringBundler.concat(
+				_objectRelationship.getCompanyId(), StringPool.POUND,
+				_objectRelationship.getObjectRelationshipId()));
+
+		ObjectRelationship objectRelationshipDefinition2 =
+			_objectRelationshipLocalService.getObjectRelationship(
+				_objectRelationship.getObjectDefinitionId2(),
+				_objectRelationship.getName());
+
+		ServiceRegistration<?>
+			objectRelationshipDefinition2ServiceRegistration =
+				serviceRegistrationMap.get(
+					StringBundler.concat(
+						objectRelationshipDefinition2.getCompanyId(),
+						StringPool.POUND,
+						objectRelationshipDefinition2.
+							getObjectRelationshipId()));
+
+		objectRelationshipDefinition2ServiceRegistration.unregister();
+
+		serviceRegistrationMap.remove(
+			StringBundler.concat(
+				objectRelationshipDefinition2.getCompanyId(), StringPool.POUND,
+				objectRelationshipDefinition2.getObjectRelationshipId()));
```

Comment:
Will the list be updated if we execute _objectRelationshipLocalService.getServiceRegistrations()?
If yes > It is better to execute the get method and assert that this ServiceRegistration is not in that map.